st2common/st2common/rbac/types.py

```diff
@@ -13,6 +13,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import itertools
```

Comment:
It looks like it's not needed for chain, but for some other itertools functionality - https://pythonhosted.org/six/#module-six.moves